public/js/app/components/AnalysisTable.jsx

```diff
@@ -10,14 +10,9 @@ const ActiveOrDisabledLink = ({ fileName, linkTitle, storageUri, roundName, vali
   return (
     <p>
       {fileName === '' ? (
-        // eslint-disable-next-line jsx-a11y/anchor-is-valid
```

Comment:
Does this make a difference for screen readers? I.e. to replace a disabled link element with a span?

Reply:
This hopefully makes better for a screen reader. A `<a>` element is meant for navigation and should always have a href. So a `<a>` without a href (or a "disabled" `<a>`) makes no sense imo. 

An alternative would be to use a `<button>` in this case (that actually can be disabled), but I think that might bit confusing i other ways. 

Some background to the eslint rule (that previously was disabled for the line):
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-is-valid.md

Reply:
Thanks for teaching me something! 🙏